docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -410,7 +409,25 @@ When you take screenshots:
 
 - Be consistent: Coordinate screenshots with the other screenshots already on a documentation page. For example, if other screenshots include the left sidebar, include the sidebar in all screenshots.
```

Comment:
Also important to only show relevant parts in the screenshot, not the whole screen.

Reply:
Good point! I think it was already mentioned in the guide. 
![screen consistency](https://user-images.githubusercontent.com/8124440/210562827-dc3ab4b2-7842-446a-bf07-f4a4d278101b.png)